src/content/integrations/@astro-communityastro-embed-youtube.md

```diff
@@ -3,7 +3,7 @@ name: "@astro-community/astro-embed-youtube"
 title: "@astro-community/astro-embed-youtube"
 description: Component to easily embed YouTube videos on your Astro site
 categories:
-  - uncategorized
+  - css+ui
```

Comment:
just checking you don't want this to be `media`? I mean, I guess you would know. 😄

Reply:
This predates the media category. I’ll need to go add more keywords after this merges 😁